### PR TITLE
[1.3] odML tables integration

### PR DESCRIPTION
**Pull request by mpsonntag**
This PR introduces plugin support of [python-odmltables](https://github.com/INM-6/python-odmltables).
- It requires odmltables version 1.1.4, which depends on INM-6/python-odmltables#13. 
- If the required version could not be found, an appropriate informative message is displayed.
- The currently supported odmltables plugins are `compare`, `convert`, `filter` and `merge`.
- Currently odmltables only supports xml files with the file ending `.odml`. Therefore, any odML document that is passed from odml-ui to one of the supported odmltables wizards, is saved as an xml file with the ".odml" file ending in the temporary odML folder also used for the terminologies and then passed along to odmltables.

Depending on how the package is installed (`python setup.py install` vs `pip install`) and which environment is used (`virtualenv` vs `conda`)
- the license file and custom icons are put to different directories
- the gtk IconTheme might not be out of the box available

Therefore this PR
- Adds an additional `conda` dependency which adds all required gtk icons.
- Adds handling paths differently if `conda` or a `virtualenv` is used.
- Changes the odmltables icons from svg to png, since the `conda` gtk environment cannot handle svgs.
- Updates `setup.py` to autogenerate the `odmlui` startup script. With the previous version it happend, that `conda` environments and `virtual` environments, when both present, were crosspolluting the shebang line of the `odml-gui` startup script rendering them unusable.

Installation has been tested for any combination of `Python2` and  `Python3` with `virtualenv` and `Anaconda` and installation option `python setup.py install` and `pip install .`.

### CODE REVIEW

**Comment by JuliaSprenger** [comment]
This version can not be run yet when using a conda environment and pip for installation. I get the following error when running the odml-gui: 

> .../odmlui/Editor.py", line 14, in <module>
>     from odml.terminology import CACHE_DIR
> ImportError: cannot import name 'CACHE_DIR'

**Comment by mpsonntag** [comment]
Good catch, its missing requirement python-odml 1.3.3 to include the CACHE_DIR...

File: odmlui/Editor.py [hunk 1]
```diff
@@ -484,6 +536,78 @@ def map_tab(self, tab):
...
+    def on_compare_entities(self, action):
+        if not self.current_tab.file_uri or self.current_tab.is_modified:
+            self._info_bar.show_info("Please validate and save "
+                                     "your document before starting odMLTables.")
+        elif self.odml_tables_available:
+            # odmltables accepts only odml xml files with an '.odml' file ending.
+            tail = os.path.split(uri_to_path(self.current_tab.file_uri))[1]
+            tmp_file = os.path.join(CACHE_DIR, ("%s.odml" % tail))
+            odml.fileio.save(self.current_tab.document, tmp_file)
+            os.system("odmltables -w compare -f %s &" % tmp_file)
```
Comment (achilleas-k) [inline odmlui/Editor.py]: Will this work if the filename has spaces in it?

File: odmlui/Editor.py [hunk 2]
```diff
@@ -484,6 +536,78 @@ def map_tab(self, tab):
...
+    def on_convert(self, action):
+        if not self.current_tab.file_uri or self.current_tab.is_modified:
+            self._info_bar.show_info("Please validate and save "
+                                     "your document before starting odMLTables.")
+        elif self.odml_tables_available:
+            # odmltables accepts only odml xml files with an '.odml' file ending.
+            tail = os.path.split(uri_to_path(self.current_tab.file_uri))[1]
+            tmp_file = os.path.join(CACHE_DIR, ("%s.odml" % tail))
+            odml.fileio.save(self.current_tab.document, tmp_file)
+            os.system("odmltables -w convert -f %s &" % tmp_file)
```
Comment (achilleas-k) [inline odmlui/Editor.py]: Same as above. If the filename has spaces it will probably fail.

File: odmlui/Editor.py [hunk 3]
```diff
@@ -484,6 +536,78 @@ def map_tab(self, tab):
...
+    def on_merge(self, action):
+        if not self.current_tab.file_uri or self.current_tab.is_modified:
+            self._info_bar.show_info("Please validate and save "
+                                     "your document before starting odMLTables.")
+        elif self.odml_tables_available:
+            # odmltables accepts only odml xml files with an '.odml' file ending.
+            tail = os.path.split(uri_to_path(self.current_tab.file_uri))[1]
+            tmp_file = os.path.join(CACHE_DIR, ("%s.odml" % tail))
+            odml.fileio.save(self.current_tab.document, tmp_file)
+            os.system("odmltables -w merge &")
```
Comment (achilleas-k) [inline odmlui/Editor.py]: Does the merge function not take file arguments?

**Review by achilleas-k — COMMENTED** [review]
Looks good, but maybe handle the filename issue for passing arguments to odmltables.

File: odmlui/Editor.py (hunk 3 above)
Comment (JuliaSprenger) [reply odmlui/Editor.py]: Yes, you can provide one or two file names using -f (--file).

Comment (mpsonntag) [reply odmlui/Editor.py]: Then we shall provide the currently open document as the first file!

**Review by achilleas-k — APPROVED** [review]

**Pull request merged**